dpcpp/base/kernel_launch_reduction.dp.hpp

```diff
@@ -612,7 +612,7 @@ void run_kernel_col_reduction_stage1(std::shared_ptr<const DpcppExecutor> exec,
     if (cols <= sg_size) {
         select_generic_col_reduction_small(
             subsubgroup_sizes(),
-            [&](int compiled_ssg_size) {
+            [cols](int compiled_ssg_size) {
                 return compiled_ssg_size >= cols ||
                        compiled_ssg_size == sg_size;
```

Comment:
I guess `sg_size` is always a compile value so it's fine like this?

Reply:
Yeah, I think so. I will see if it compiles. I was too lazy to try and compile it myself with dpcpp :P